jobs/MasterCI/MasterCI

```diff
@@ -66,7 +66,9 @@ node{
             }
         } catch(error){
             // trigger freeze procedure
-            build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
+            if (currentBuild.result == "FAILURE"){
+                build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
```

Comment:
It is to avoid merge freeze when only publish phase failed but master ci is marked as green